src/actions/application.js

```diff
@@ -78,7 +78,38 @@ function* loadContentTypes() {
   }
 }
 
+/**
+ * Gets all available action types.
+ */
+export const ACTIONS_REQUESTED = 'ACTIONS_REQUESTED';
```

Comment:
I considered the list of available actions to be some form of global information. Not sure though how useful it is to share this.